src/ImageRunModal.jsx

```diff
@@ -52,6 +52,14 @@ const units = {
     },
 };
 
+// healthchecks.go HealthCheckOnFailureAction
+const HealthCheckOnFailureActionOrder = [
+    { value: 0, label: _("No action") },
+    { value: 3, label: _("Restart") },
+    { value: 4, label: _("Stop") },
+    { value: 2, label: _("Force stop") },
```

Comment:
This looks weird -- what' is value 1? And can we keep it sorted numerically? It looks like something went wrong here.

Reply:
This is just wrong now, we can keep it numerically sorted but then need something else to sort it correctly for the UI.

Reply:
Oh, that's the order in which we show it in the UI, I see. Keep it then, yes.